yarn.lock

```diff
@@ -5863,6 +5867,15 @@ react-error-overlay@^4.0.0:
   version "4.0.0"
   resolved "https://registry.yarnpkg.com/react-error-overlay/-/react-error-overlay-4.0.0.tgz#d198408a85b4070937a98667f500c832f86bd5d4"
 
+react-helmet@^5.2.0:
```

Comment:
why is this thing here? Ideally, it shouldn't be since I merged the other PR (`yarn lock` is a pain)

Reply:
I'm sorry, new to github. What can I do to solve it?

Reply:
Ah, never mind, I did it myself

Reply:
mercí beaucoup :)